sklearn/gaussian_process/gpr.py

```diff
@@ -140,8 +141,20 @@ def __init__(self, kernel=None, alpha=1e-10,
         self.copy_X_train = copy_X_train
         self.random_state = random_state
 
+    @property
+    @deprecated("Attribute rng was deprecated in version 0.19 and "
+                "will be removed in 0.21. Use '_rng' instead")
+    def rng(self):
+        return self._rng
+
+    @property
+    @deprecated("Attribute y_train_mean was deprecated in version 0.19 and "
+                "will be removed in 0.21. Use '_y_train_mean' instead")
```

Comment:
Do not tell the user to use a private attribute. Just say it's deprecated and will be removed.